game-app/game-core/src/main/java/games/strategy/triplea/attachments/AbstractRulesAttachment.java

```diff
@@ -217,17 +211,17 @@ private void setRounds(final Map<Integer, Integer> value) {
 
   @VisibleForTesting
   public Map<Integer, Integer> getRounds() {
-    return turns;
+    return getMapProperty(turns);
   }
 
   private void resetRounds() {
     turns = null;
   }
 
-  protected boolean checkTurns(final GameState data) {
+  protected boolean checkRounds(final GameState data) {
     final int turn = data.getSequence().getRound();
-    for (final int t : turns.keySet()) {
-      if (turn >= t && turn <= turns.get(t)) {
+    for (final int t : getRounds().keySet()) {
```

Comment:
Instead of using `keySet()` and then calling `get()` with the key, I'd recommend using `entrySet()` instead

Reply:
Done.